ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py

```diff
@@ -137,6 +137,51 @@ def test_torch_roll(
     )
 
 
+# meshgrid
+@handle_frontend_test(
+    fn_tree="torch.meshgrid",
+    dtypes_and_tensors=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float").filter(
+            lambda x: "bfloat16" not in x[0]
+        ),
+        num_arrays=st.integers(min_value=1, max_value=5),
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=2,
+        max_dim_size=5,
+        shared_dtype=True,
+    ),
+    indexing=st.sampled_from(['ij', 'xy']),
+)
+def test_torch_meshgrid(
+    *,
+    dtypes_and_tensors,
+    indexing,
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
+):
+    dtypes, tensors = dtypes_and_tensors
+    if isinstance(dtypes, list):

```

Comment:
the if else statement here is actually not needed as the output is always a list, just keep the `kwargs = {
        f"tensor{i}": np.array(tensor, dtype=dtypes[i])
        for i, tensor in enumerate(tensors)
    }`